src/services/user/model/user.schema.js

```diff
@@ -67,6 +68,11 @@ userSchema.index({ schoolId: 1, roles: -1 });
 // maybe the schoolId index is enough ?
 // https://ticketsystem.schul-cloud.org/browse/SC-3724
 
+if (Configuration.get('FEATURE_TSP_ENABLED') === true) {
+	// to speed up lookups during TSP sync
+	userSchema.index({ 'sourceOptions.$**': 1 });
```

Comment:
Wenn der index in der DB einmal gesetzt ist, bekommt man ihn durch das setzten von false nicht mehr raus.
Evtl. noch ein index Sync im Fall von false?

Reply:
der Fall wird niemals eintreten in Production und in dev ist das mit autoindexing erledigt

Reply:
damit es in den anderen instanzen nicht durch spätere Migrationen aus versehen angeschalten wird

Reply:
ok